src/AppRouter.js

```diff
@@ -48,10 +48,6 @@ const AppRouter = () => (
             elements={elements}
             initialState={{ ...match.params }}
             popupComponents={POPUP_CONF}
-            cartaroUrl={process.env.REACT_APP_CARTARO_URL}
-            geoServerUrl={process.env.REACT_APP_GEOSERVER_URL}
-            vectorTilesKey={process.env.REACT_APP_VECTOR_TILES_KEY}
-            vectorTilesUrl={process.env.REACT_APP_VECTOR_TILES_URL}
```

Comment:
We will still need this Ithink when the AppRouter will exits only in gitlab

Reply:
Yeah, we can add them again, if needed. Just wanted to move the default value into TrafimageMaps.